Bio/SubsMat/__init__.py

```diff
@@ -260,7 +260,7 @@ def sum(self):
         return result
 
     def format(self, fmt="%4d", letterfmt="%4s", alphabet=None,
-               factor=1, non_sym=None, full=False):
+               non_sym=None, full=False):
```

Comment:
I'm guessing you didn't mean to remove the format argument yet (seeing as you check for it later with a warning if not one)?

Reply:
Fixed, thanks for catching this

Reply:
I still see the removal of ``factor=1`` in the arguments to ``def format(self, ..):`` one lines 263 and 264?

Reply:
Yes it was removed, as it's not being used anywhere, and the format() function itself was added only yesterday.

Reply:
Ah - in that case, no problem 👍